### ISSUE TITLE
What next?

### ISSUE THREAD

**Issue by simlu** [simlu/voxelshop#167]
So I'm thinking, rather than implementing features like crazy I will fix on documentation and video tutorials and make sure that people who want to utilize VoxelShop for other stuff can do it nicely (e.g. StoneHearth, Blender, Unity, etc).

A lot of people don't even know the basic capabilities of VoxelShop...

What do you guys think? Also what features do you think are really important to tackle next and why, i.e. what do you think is the benefit.

**Comment by MrTopom** [comment]
I'll continue the wiki with basic functionalities, I then will learn same time.

Related to features, I'll need for my current project the following things:
- Basic shape command (line, extrude, ...)
- Procedural generation
- Batch export (command line)
- Little plugin to generate things from the procedural generation (Bounding box, ...)

**Comment by simlu** [comment]
@MrTopom Updating the wiki would be great! Really need to get it into a more complete state...

**Comment by kevzettler** [comment]
I think you have the right idea. User education, documentation, and adoption should be a focus. Once you grow the user base you Ideally get:

* more Feedback and feature requests from users that help guide roadmap
* more Open source contributors. 
* more virality, artists using and talking about voxel shop

My thoughts are skeletal animation could be the killer feature that sets VoxelShop apart from the crowd. expanded on this idea here: https://github.com/simlu/voxelshop/issues/168#issuecomment-258488126

**Comment by simlu** [comment]
Great. Let's push for 1.8.X and then get some promotion going. Would be great to have a the wiki in an acceptable state for that. Lots of work to do :)

https://github.com/simlu/voxelshop/milestone/5

**Comment by kevzettler** [comment]
In addition to the User education and this is also relevant to #168 I would add that a StackExchange like http://blender.stackexchange.com/ You might have to pay for stackexchange so a reddit /r/voxelShop would work too

For Q and A would be great. I"m not a fan of video education content as it is not Search engine friendly.

**Comment by simlu** [comment]
Good point. There is an FAQ section in the wiki, but it needs filling...

The think with VoxelShop is that it is much easier to show than to explain features. And everything would be accompanied by text. So I don't think it's an argument against video.

**Comment by kevzettler** [comment]
good points

**Comment by simlu** [comment]
Reading over this again... It seems performance is the most important feature all users want. Virtually no-one wants to create small voxel models (what Voxelshop was originally designed for). So The focus is on improving that next...
